public/locales/en/translation.json

```diff
@@ -269,7 +269,7 @@
       "details": "Please fill in your phone number to begin",
       "logo-alt-text": "telegram logo",
       "alerts": {
-        "code-sent": "A verification code has been sent to your phone. Check code in your sms or email inbox",
+        "code-sent": "A verification code has been sent to your telegram. Check code in your sms or email inbox",
```

Comment:
This should be telegram account or telegram App and the following text should reflect the first sentence